### DISCUSSION TITLE
Support for depth estimation task evaluation methods including RMSE 支持包括 RMSE 在内的深度估计任务评估方法

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#607] [MMSegmentation]
# Support for depth estimation task evaluation methods including RMSE 
 支持包括 RMSE 在内的深度估计任务评估方法 

- [Reference implementation](https://github.com/zhyever/Monocular-Depth-Estimation-Toolbox/blob/main/depth/core/evaluation/metrics.py#L34)

[参考实现](https://github.com/zhyever/Monocular-Depth-Estimation-Toolbox/blob/main/depth/core/evaluation/metrics.py#L34)

## Task Information

**Repository**：MMSegmentation

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：SheffieldCao

**Claim Time**：2023年10月25日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by SheffieldCao** [comment]
Claim the mission
